src/moving_platform.hpp

```diff
@@ -17,18 +19,70 @@ namespace MovingPlatform {
  * With this class a platforms movement can be set with ease
  */
 class Platform {
-    //? & leftMotor;
-    //? & rightMotor;
+  private:
+    UARTLib::UARTConnection &serialCon;
+    Qik2S12V10 motorController;
+    hwlib::pin_in &hallSensorPinA;
+    hwlib::pin_in &hallSensorPinB;
+
     int16_t currentSpeedLeft, currentSpeedRight;
     int16_t wheelsize;
     int16_t smoothing;
     int16_t speed;
+    static constexpr uint32_t rotationsPerDegree = 24;
+    static constexpr uint8_t radiusMm = 62;
+    static constexpr uint16_t circumferenceMm = 2 * 3.14159265359 * radiusMm;
+    static constexpr uint16_t umPerDegree = (circumferenceMm * 1000) / 360;
+    static constexpr uint16_t countsPerRotation = 3200;
 
   public:
     enum class Direction { Forward, Backward };
 
-    // Platform(? & leftMotor, ? & rightMotor);
-    Platform();
+  private:
+    /**
+     * @brief Moves the platform forward.
+     *
+     * Move the platform forward for *distanceMilliMeters*.
+     *
+     * @param[in] distanceMilliMeters The amount of millimeters the platform should move.
+     */
+    void movePlatformForward(int32_t distanceMilliMeters);

```

Comment:
Millimetres is 1 word, so: distanceMillimetres.

Reply:
Changed to distanceMillimeters